### DISCUSSION TITLE
Snippet: Hiding a block?

### DISCUSSION THREAD

**Discussion by paaljoachim** [WordPress/developer-blog-content#354] [Topic Ideas]
Hey

I am wondering if it would be possible to hide a block?
On occasion I just want to hide a block and not remove it. To make comparisons.

https://github.com/WordPress/gutenberg/issues/13373

A very very light version of @ndiego Nick's plugin here: https://wordpress.org/plugins/block-visibility/

When I think through it, it quickly becomes fairly complex.
Adding an option to the 3 dot toolbar dropdown link to where one can hide/unhide a block.
Hiding could make it grey on the backend and hidden on the frontend.

**Comment by ndiego** [comment]
@paaljoachim I converted this to a discussion, since that's how topic ideas start for the Dev Blog. Are you looking for a tutorial on this topic?

**Comment by paaljoachim** [comment]
@ndiego Nick yes. That would be great!

I could then post a link to a tutorial on the old now closed issue I made a bunch of years ago.

**Reply by ndiego** [reply]
Ok, cool. I think the framing of such an article will need to be a bit more generic than specifically how to hide blocks. More about the mechanics of building a plugin that registers a custom attribute (in this case for hiding the block), adding an option to the Options menu, and then modifying the frontend render function. 

As an aside, this functionality might be coming to Core. You can follow the discussion here: https://github.com/WordPress/gutenberg/discussions/67661

**Reply by paaljoachim** [reply]
O yes! Perfect timing with adding this discussion. As it just shows the additional need for having this functionality in core.
Thanks Nick!

**Comment by bph** [comment]
Hi @paaljoachim As Hiding Blocks has been built into WordPress 6.9, with refinement for viewport-related display options,  would you be ok with us closing this topic "as outdated"

**Reply by paaljoachim** [reply]
Yes. Good idea Birgit @bph .

**Comment by paaljoachim** [comment]
Outdated
